### DISCUSSION TITLE
Does v2 documentation mentions --cwd option?

### DISCUSSION THREAD

**Discussion by pustomytnyk** [yarnpkg/berry#2448] [Q&A]
I can't find the mention of `--cwd` flag in v2 docs. The only mention I see is for `ignoreCwd`. 

v1 has general CLI docs on https://classic.yarnpkg.com/en/docs/cli/, but v2 doesn't have such page (https://yarnpkg.com/cli redirects to https://yarnpkg.com/cli/install).

**Comment by pustomytnyk** [comment]
@merceyz Are you sure that's not a documentation issue? https://github.com/yarnpkg/berry/issues/2447

**Reply by peterp** [reply]
command works, so must be a documentation issue.
